### PR TITLE
Hosted Happs: Add v0.14.4-alpha1 HoloFuel for testing

### PR DESCRIPTION
**Pull request by alastairong**
Adding the version used by Matt for compatibility with his UI. Doesn't need to be reviewed yet. 

Need to update from develop once https://github.com/Holo-Host/holo-update-conductor-config/pull/6 is added to holo-nixpkgs

### CODE REVIEW

File: overlays/holo-nixpkgs/dna-packages/default.nix [hunk 1]
```diff
@@ -16,6 +16,12 @@ let
     sha256 = "159g4d0fhmb4kfi7v4ndizamj6ajfmxxv57ylzkr9q8sdyjb5l8i";
   };
 
+  hosted-holofuel = fetchurl {
+    url = "https://holo-host.github.io/holofuel/releases/download/v0.14.4-alpha1/holofuel.dna.json";
+    name = "testfuel.dna.json";                                         # Is arbitrarily changing this name ok?
```
Comment (peeech) [inline overlays/holo-nixpkgs/dna-packages/default.nix:21]: In current implementation name has to match file name form the line above

File: overlays/holo-nixpkgs/dna-packages/default.nix [hunk 2]
```diff
@@ -16,6 +16,12 @@ let
...
+    sha256 = "1bzrmw3v0kf6q76s6h56cyxv5axjcjd0axpkbkp07y7cdd8s356r";    # Does the hash change if the name is changed?
```
Comment (peeech) [inline overlays/holo-nixpkgs/dna-packages/default.nix:22]: It does not

File: overlays/holo-nixpkgs/dna-packages/default.nix (hunk 1 above)
Comment (alastairong) [reply overlays/holo-nixpkgs/dna-packages/default.nix:21]: Ok does name have to be unique?

Comment (peeech) [reply overlays/holo-nixpkgs/dna-packages/default.nix:21]: Does not have to be unique. you can call as many dna files `holofuel.dna.json` as you want.

Comment (alastairong) [reply overlays/holo-nixpkgs/dna-packages/default.nix]: Addressed in `ba242b3`

**Comment by mjbrisebois** [comment]
Closing in favor of https://github.com/Holo-Host/holo-nixpkgs/pull/535.  All relevant changes have been implemented in new PR.

**Pull request closed without merging**